be/src/exprs/cast_expr.h

```diff
@@ -149,4 +151,30 @@ class CastStructExpr final : public Expr {
     std::vector<std::unique_ptr<Expr>> _field_casts;
 };
 
+/**
+ * Cast other type to string without float, double, string
+ */
+struct CastToString {
+    template <typename Type, typename ResultType>
+    static std::string apply(const Type& v) {
+        if constexpr (IsDate<Type> || IsTimestamp<Type> || IsDecimal<Type>) {
+            // DateValue, TimestampValue, DecimalV2
```

Comment:
DecimalV2 is deprecated,   use decimal v3,  DecimalV3Cast::to_string to convert.

Reply:
actually decimalV3 is supported but not in this code path, so don't need to worry about it